PublicMapping/districtbuilder-classic
# Make Geoserver admin user password configurable #155082923

## Overview

The Geoserver admin username/password is by default admin/geoserver. This is easily discoverable online, so we want to change it as part of the setup process to what's defined in `.env`.

### Checklist


### Notes

The script assumes that the password is 'geoserver' for it to be reset. As such, the idea is for this password to only need to be reset once (subsequent resets won't do anything).

As with other components in the system, if you change passwords in `.env` without starting over, things may not work right or steps may have to be repeated. This "foward only" assumption seems to have been in place up until now, so I don't think it hurts to keep going in that same vein.

## Testing Instructions

 * In `.env`, change `MAP_SERVER_ADMIN_PASSWORD` to something besides 'geoserver'. Delete `MAP_SERVER_ADMIN_USER` from `.env`.
 * Run `./scripts/update` (password will be reset based on `.env` by running this script)
 * `./scripts/server`
 * Go to http://localhost:8080/geoserver/web/ and log in using the new password.
 * `vagrant ssh` and `docker-compose exec -T django ./manage.py setup config/config.xml -G`
 * Configuration of Geoserver should succeed as before

Closes #155082923
